src/test/java/org/apache/sling/jcr/repoinit/PrincipalBasedAclTest.java

```diff
@@ -725,16 +762,16 @@ public void testRemovePrincipalMismatch() throws Exception {
                 + "end";
         U.parseAndExecute(setup);
         U.parseAndExecute("create service user otherSystemPrincipal");
+        assertPolicy(getPrincipal(U.username), U.adminSession, 1);
 
-        try {
-            setup = "set principal ACL for otherSystemPrincipal\n"
-            + "remove jcr:write on " + path + "\n"
-            + "end";
-            U.parseAndExecute(setup);
-            fail("Expecting REMOVE to fail");
-        } catch(RuntimeException rex) {
-            assertRegex(REMOVE_NOT_SUPPORTED_REGEX, rex.getMessage());
```

Comment:
Same comment as above for a test that goes away

Reply:
see above.